qiskit/providers/backend.py

```diff
@@ -229,3 +241,445 @@ class can handle either situation.
+        props = self.qubit_properties(qubits)
+        if isinstance(props, QubitProperties):
+            if props.t1 is None:
+                raise ValueError(f"T1 is not defined for qubit {qubits}")
+            return props.t1
+        out_list = []
+        for index, prop in enumerate(props):
+            if prop.t1 is None:
+                raise ValueError(f"T1 is not defined for qubit {qubits[index]}")
+            out_list.append(prop.t1)
+        return np.array(out_list)
+
+    def t2(self, qubits: Union[int, List[int]]) -> Union[float, np.array]:
+        """Return the T2 time of a given qubit
+
+        Args:
+            qubits: The qubit index or indices to get the T2 time for. If
+                a list is specified the output will be a list with the
+                T2 time for the specified qubits in the same order.
+
+        Returns:
+            t2: the T2 time for the specified qubit(s) in seconds
+
+        Raises:
+            NotImplementedError: if the backend doesn't support querying the
+                qubit properties for a qubit
+            ValueError: if T2 is not defined for the specified qubit
+        """
+        props = self.qubit_properties(qubits)
+        if isinstance(props, QubitProperties):
+            if props.t2 is None:
+                raise ValueError(f"T2 is not defined for qubit {qubits}")
+            return props.t1
+        out_list = []
+        for index, prop in enumerate(props):
+            if prop.t2 is None:
+                raise ValueError(f"T2 is not defined for qubit {qubits[index]}")
+            out_list.append(prop.t2)
+        return np.array(out_list)
+
+    @property
+    def dt(self) -> Union[float, None]:
+        """Return the system time resolution of input signals
+
+        This is required to be implemented if the backend supports Pulse
+        scheduling.
+
+        Returns:
+            dt: The input signal timestep in seconds. If the backend doesn't
+            define ``dt`` ``None`` will be returned
+        """
+        return self.target.dt
+
+    @property
+    def dtm(self) -> float:
+        """Return the system time resolution of output signals
+
+        Returns:
+            dtm: The output signal timestep in seconds.
+
+        Raises:
+            NotImplementedError: if the backend doesn't support querying the
+                output signal timestep
+        """
+        raise NotImplementedError
+
+    @property
+    def meas_map(self) -> List[List[int]]:
+        """Return the grouping of measurements which are multiplexed
+
+        This is required to be implemented if the backend supports Pulse
+        scheduling.
+
+        Returns:
+            meas_map: The grouping of measurements which are multiplexed
+
+        Raises:
+            NotImplementedError: if the backend doesn't support querying the
+                measurement mapping
+        """
+        raise NotImplementedError
+
+    @property
+    def instruction_schedule_map(self):
```

Comment:
What would a circuit simulators return here? None?

Reply:
It will return an empty `InstructionScheduleMap` unless we have `AerSimulator.from_backend()` set the calibration field for instructions in the `Target` object. Then it would return the `InstructionScheduleMap` from that.